### ISSUE TITLE
Todo in near future

### ISSUE THREAD

**Issue by sashaegorov** [sashaegorov/foundation-player#8]
@isitrita here is the random list of stuff to do.
Let's choose what have to be done first.

**Random unsorted list**:
- [ ] _FIXME_ **IE9/10** completely sucks now :-(
- [x] _FIXME_ **FF** completely sucks now :-(
- [x] _FIXME_ **Safari** doesn't start player
- [ ] _FEATURE_ Mobile actions like touch and etc.
- [ ] _FEATURE_ Mobile "first" responsive :-(
- [ ] _FEATURE_ Error handling, at least media-file download error
- Check is it possible to get meta information from audio

**Nice to have**:
- Pressed state for buttons
- Highlight table of contents on progress bar e.g. dots or bars

**Items related to multi-item player**:
- Next/Previous buttons when necessary
- Buffering option for playlist items and single media-file
- Ability to manage many `<audio>` elements via playlist

Done:
- [x] _FEATURE_ ~~Loading indicator~~
- [x] _FIXME_ ~~aN:aN in statuses while loading... related to previous~~
- [x] _FIXME_ ~~Fix Safari quirks for buttons hover state~~
- [x] ~~API Method to navigate to timestamp e.g. '02:10'~~
- [x] 39038c3ada87bdba278970e5d30e9d1c5d158cbc ~~_FIXME_ Dragging stops when cursor go outside dragbar~~
- [x] ~~Shut others when it starts when we have many players on page~~

**Comment by isitrita** [comment]
HI, 
this feature is done:
 ~~Shut others when it starts~~

I will take `Highlight table of contents on progress bar e.g. dots or bars`
